### DISCUSSION TITLE
Change PSC Term End Date and Add Shawn Crawley to PSC Member List

### DISCUSSION THREAD

**Discussion by swainn** [tethysplatform/tethys#915] [By-Laws Amendment]
All,

Please take a moment to review the following proposed changes to the [Tethys Platform project By-Laws document](http://www.tethysplatform.org/project-steering-committee) and comment with suggestions or concerns. This public comment period will be open until February 2nd, 2023 at Midnight (UTC).

To see the changes, see this PR:
https://github.com/tethysplatform/tethysplatform.github.io/pull/8

Please post comments on this discussion.

--
Tethys Platform Project Steering Committee

**Comment by sdc50** [comment]
@swainn I added an [additional commit](https://github.com/tethysplatform/tethysplatform.github.io/pull/8/commits/e2519258ff547d149f88274fb8227f829e53c80d) to this PR that just changes the years that are shown as an explanation of when elections will be held for each term. 

E.g:

![image](https://user-images.githubusercontent.com/6454472/215820003-ad4e492c-20d3-4a12-8270-6ae153df73fd.png)

**Comment by swainn** [comment]
These changes will be merged.
